sunpy/net/vso/vso.py

```diff
@@ -61,12 +61,37 @@ def check_connection(url):
         return None
 
 
+def check_cgi_connection(url):
+    """
+    At the moment there is no way to make an "are you alive" request to the
+    cgi, so we just hit it with a HTTP get and it gives us back a 411 response.
+    This is weird enough that it probably satisfies us for this check.
+    """
+    try:
+        return urlopen(url).getcode() == 411
```

Comment:
```suggestion
        return urlopen(url, timeout=10).getcode() == 411
```

We probably don't want to wait the 5 minute default timeout?